package.json

```diff
@@ -23,6 +23,9 @@
   "stylelint": {
     "extends": "stylelint-config-gds/scss"
   },
+  "dependencies": {
+    "jquery": "1.12.4"
+  },  
```

Comment:
I did try that earlier but can't remember why I changed it back, anyway, just changed it again and this happens:
```
Sprockets::FileNotFound: couldn't find file 'jquery/dist/jquery' with type 'application/javascript'
Checked in these paths: 
  /app/assets/config
  /app/assets/images
  /app/assets/javascripts
  /app/assets/stylesheets
  /vendor/assets/javascripts
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/app/assets/config
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/app/assets/images
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/app/assets/javascripts
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/app/assets/stylesheets
  /usr/local/bundle/gems/chartkick-4.1.3/vendor/assets/javascripts
  /usr/local/bundle/gems/actionview-7.0.2.4/lib/assets/compiled
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/node_modules/govuk-frontend/govuk/assets/images
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/node_modules/govuk-frontend/govuk/assets/fonts
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/node_modules/govuk-frontend
  /usr/local/bundle/gems/govuk_publishing_components-29.6.0/node_modules
  /node_modules
[0m[91m/app/assets/javascripts/application.js:1
```

Bit confused about this.

Reply:
Ah so the problem is that Rails 7 has stopped installing yarn dependencies automatically as part of `assets:precompile`. Since we don't have a step in our deployment to replicate that currently I've added this in as an enhancement to the `assets:precompile` task in [c3500e3](https://github.com/alphagov/release/pull/960/commits/c3500e36540a71051a83148046345a3f5514b6de)

I can't work out where it was finding jquery from in `//= require jquery` though 😕 

Anyway - I've dropped a couple of commits on to sort out the problem.